index.js

```diff
@@ -45,15 +45,25 @@ class Logger {
       json: require('./lib/json'),
       cli: require('./lib/cli'),
     };
+    if (this.config.reporters) {
```

Comment:
There isn't one, they are synonyms. We had a previous unit test that was using the term 'plugins' for 'reporters' so I kept supporting it for backwards compatibility.  We can dump one or the other term if you want.